Backend/ManagerLayer/UserManagement/UserManagementManager.cs

```diff
@@ -15,28 +10,21 @@ public class UserManagementManager
     {
         private IPasswordService _passwordService;
         private IUserService _userService;
-        private readonly DatabaseContext _db;
 
         public UserManagementManager()
         {
             _userService = new UserService();
         }
 
-        public UserManagementManager(DatabaseContext db)

```

Comment:
This should all be reverted back

Reply:
pass in _db into the instance constructor.

Reply:
Yes, but it needs to be done in all the Managers/Services and that isn't really the point of this pr. That should probably be done in a separate branch after this code is merged so this merge is not pushed back any further and so it doesn't grow any larger.